apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -113,9 +114,10 @@ class DanceVisualizationColumn extends React.Component {
         {!this.props.isShareView && (
           <AgeDialog turnOffFilter={this.turnFilterOff} />
         )}
-        {(this.props.over21 || this.props.userType === 'teacher') && (
-          <HourOfCodeGuideEmailDialog isSignedIn={isSignedIn} />
-        )}
+        {(this.props.over21 || this.props.userType === 'teacher') &&
```

Comment:
This has nothing to do with this PR but is our plan to show this to teachers who have already created an account?

Reply:
Yes- there are two different versions of the component: one for signed in teachers and the other for signed out users who've selected 21+

Reply:
Ah got it that makes sense!